packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -439,6 +439,11 @@ export interface IMenuItemStyles extends IButtonStyles {
   iconColor: IStyle;
 
   /**
+   * Default color style for checkmark icons.
```

Comment:
should remove mentions of color from here, since an IStyle could be any styles, not just colors